src/NuGet.Indexing/NuGetQuery.cs

```diff
@@ -89,6 +93,15 @@ private static Query ConstructQuery(Dictionary<QueryField, HashSet<string>> clau
                             }
                         }
 
+                        if (packageTypes != null)
```

Comment:
I don't think you need this, unless we want to support querying the package type in free-form queries? (e.g. when searching for "dotnettools", do we want to also search our package type field? If yes, keep this. If no, drop the entire if and its body :-))